x-pack/platform/plugins/shared/fleet/server/services/epm/packages/knowledge_base_index.ts

```diff
@@ -0,0 +1,229 @@
+/*
```

Comment:
unit tests for this file or do you expect the functionality to be covered by the integration test?